build.sbt

```diff
@@ -14,6 +14,8 @@ ThisBuild / developers ++= List(
   tlGitHubDev("baccata", "Olivier Mélois")
 )
 
+ThisBuild / githubWorkflowJavaVersions := Seq(JavaSpec.temurin("17"))
```

Comment:
This is because I bring an eclipse jgit version that is built for Java 11. Not really intentional, but probably not a big deal to use newer JDK on CI.